### PR TITLE
Logging support

### PR DESCRIPTION
**Pull request by zippy1981**
Event source and eventid 1 needs to be added to installer, but here is my stab at logging support. I updated all the packages. Definitely need to update nlog to make this one work.

### CODE REVIEW

File: RetailCoder.VBE/UI/DockablePresenterBase.cs [hunk 1]
```diff
@@ -28,13 +32,18 @@ private Window CreateToolWindow(IDockableUserControl control)
...
+            catch (COMException exception)
             {
-                toolWindow = _vbe.Windows.CreateToolWindow(_addin, _DockableWindowHost.RegisteredProgId,
-                    control.Caption, control.ClassId, ref userControlObject);
+                var logEvent = new LogEventInfo(LogLevel.Error, _logger.Name,  "Error Creating Control");
+                logEvent.Exception = exception;
+                logEvent.Properties.Add("EventID", 1);
+
+                _logger.Error(logEvent);
+                throw;
```
Comment (ckuhn203) [inline RetailCoder.VBE/UI/DockablePresenterBase.cs]: I don't mind the logging, but I'm not okay with removing the retry until we understand the problem and actually fix it.

**Comment by ckuhn203** [comment]
For posterity, only the NLog and Moq packages were updated.

**Comment by ckuhn203** [comment]
The build failed, but it's not you. It's the features we've not implemented yet. 

https://ci.appveyor.com/project/Rubberduck/rubberduck-3v9qv/build/1.4.373-next

**Comment by zippy1981** [comment]
Ok so if I put the retry back and add the event ID to the installer is this mergable?

**Comment by ckuhn203** [comment]
Yeah. Looks good to me, but I'd like @retailcoder to give it a once over. He's the one who installed NLog to begin with.

**Comment by retailcoder** [comment]
@zippy1981 If you know how to add the event ID, by all means go ahead and do it - I'd have to do some research, as I never logged to the EventLog... but I love the idea, thanks for this PR!

**Pull request merged**